packages/test/end-to-end-tests/src/test/.mocharc.js

```diff
@@ -9,14 +9,21 @@ const testDriver = process.env.FLUID_TEST_DRIVER ? process.env.FLUID_TEST_DRIVER
 const packageDir = `${__dirname}/../..`;
 const testPackagesDir = `${packageDir}/..`;
 
-const config ={
+const requiredModules = [
+    `${testPackagesDir}/mocha-test-setup`, // General mocha setup e.g. suppresses console.log
+    `${testPackagesDir}/test-drivers`, // Inject implementation of getFluidTestDriver, configured via FLUID_TEST_DRIVER
+];
+
+if (process.env.FLUID_DI_ROOT && process.env.FLUID_DI_LOGGERPKG) {
```

Comment:
we should consider just making this a single var, like FUILD_TEST_LOGGER_PKG_PATH

Reply:
Haha yeah I have it that way in my next branch :P